helpers/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
Please move this into the `test` folder, and rename it to utils. 
This'll keep the root clean, and I think utils is a bit more common in truffle.